pkg/zson/record.go

```diff
@@ -89,7 +91,8 @@ func NewRecordZvals(d *Descriptor, vals ...zval.Encoding) (t *Record, err error)
 	var ts nano.Ts
 	if col, ok := d.ColumnOfField("ts"); ok {
 		var err error
-		ts, err = nano.Parse(vals[col])
+		//XXX this needs to call Decode
+		ts, err = zeek.DecodeTime(vals[col])
```

Comment:
Isn't zeek.DecodeTime for machine types? With this work?

Reply:
Whoops, nevermind.

Reply:
The comment should be removed.  But yes, zvals need to be decoded and the vals argument is presumably a bunch of zvals that have already been encoded.